### PR TITLE
add: newly designed footer

### PR DESCRIPTION
**Pull request by hannahouazzane**
Create footer as per designs
Note that the current PR does not exclude the footer from the home page. 

Footer should look like designs below:
Screenshot 2022-11-08 at 21 14 07

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -2,137 +2,39 @@ import React from 'react'
...
-                </InternalLink>
-              </Text>
-            </div>
-          </div>
-        </footer>
-      </Text>
-    </div>
+  <footer className="footer-container">
+    <ConstrainedWidth paddingTop="2rem">
+      {/* container */}
```
Comment (rosejbon) [inline src/components/Footer/index.js:11]: Just a little comment left in :)

**Review by rosejbon — APPROVED** [review]
Looks good to me

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -2,137 +2,39 @@ import React from 'react'
...
+        width: 100%;
+        background-color: ${c_NEON_PURPLE};
       }
 
-      .twitterlogo {
-        display: inline-block;
-        height: 1rem;
+      .flex-container {
+        display: flex;
+        justify-content: space-between;
```
Comment (Nikomus) [inline src/components/Footer/index.js:31]: Maybe use align-items here to pull the link in line with the logo?

File: src/components/Layout/ConstrainedWidth.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ const ConstrainedWidth = ({ children }) => (
         margin-right: auto;
         border-radius: 0 0 38px 0;
         padding-bottom: 2rem;
+        ${paddingTop ? `padding-top: ${paddingTop};` : ''}
```
Comment (Nikomus) [inline src/components/Layout/ConstrainedWidth.js:17]: Is this border radius in use here? I don't think I would expect this component to provide any vertical padding, it's been extracted as a utility to control the page width.

File: src/components/Footer/index.js (hunk 2 above)
Comment (hannahouazzane) [reply src/components/Footer/index.js:31]: Sounds good!

File: src/components/Layout/ConstrainedWidth.js (hunk 3 above)
Comment (hannahouazzane) [reply src/components/Layout/ConstrainedWidth.js:17]: I think I also used this component as the Project cards which have one curved edge. I could create a separate component for the project cards ?

File: src/components/Footer/index.js (hunk 1 above)
Comment (hannahouazzane) [reply src/components/Footer/index.js:11]: TY :))

**Pull request merged**